packages/babel-preset-react-server/package.json

```diff
@@ -29,5 +31,8 @@
     "babel-plugin-transform-es2015-template-literals": "^6.3.13",
     "babel-plugin-transform-object-rest-spread": "^6.3.13",
     "babel-preset-react": "^6.5.0"
+  },
+  "devDependencies": {
+    "rimraf": "^2.2.8"
```

Comment:
The downside of pulling them to the monorepo root is reduced portability.  In particular, dependencies that define a `bin` script have the script added to `node_modules/.bin`, which is available to `package.json` scripts hash.  If I were to pull this dep to the monorepo root, I'd have to specify a path to the bin script (i.e. `../../node_modules/.bin/rimraf`), which would break our scripts for developers on windows machines.